Makefile

```diff
@@ -11,4 +11,4 @@ lint-prepare:
 goimports-prepare:
 	@echo "Preparing goimports"
 	go get golang.org/x/tools/cmd/goimports
-	go mod tidy
\ No newline at end of file
+	go mod tidy
```

Comment:
@slzhffktm did u have another `editorconfig` that overrides the one in this repo? I saw a lot of trailing_new_lines missing

Reply:
oh my, sorry I accidentally pushed this. But I don't think so, bcs the there is also gofmt in pre-commit.